#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_extraqualifiers.dpatch by  <richardaburton@hotmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove extra qualifiers (for gcc 4.1)

@DPATCH@

diff -ur kicad-0.0.20060626.orig/pcbnew/track.cpp kicad-0.0.20060626/pcbnew/track.cpp
--- kicad-0.0.20060626.orig/pcbnew/track.cpp	2006-04-19 16:31:18.000000000 +0100
+++ kicad-0.0.20060626/pcbnew/track.cpp	2006-06-26 16:36:14.000000000 +0100
@@ -17,7 +17,7 @@
 	TRACK * RefTrack;
 
 public:
-	TSTSEGM::TSTSEGM(TRACK * Father)
+	TSTSEGM(TRACK * Father)
 		{
 		Pback = Pnext = NULL;
 		RefTrack = Father;
